manifests/init.pp

```diff
@@ -173,6 +177,10 @@
 #   Integer.  The Redis instance DB to use/select
 #   Default: 0
 #
+# [*redis_sentinels*]
+#   Array. Redis Sentinel configuration and connection information for one or more Sentinels
```

Comment:
As above, explicitly state it is an `Array of hashes`. Fits with elsewhere in file where it explicitly refers to `Array of strings` eg subscriptions.